resources/views/home.blade.php

```diff
@@ -58,11 +60,116 @@
     </div>
   </div>
 
-@include('layouts/navbarCaasLogout')
-@section('navcaaslogout')
-@endsection
-
-<section id="main-nim">
+  @include('layouts/navbarCaasLogout')
+  @section('navcaaslogout')
+  @endsection
+  <section id="landing-section" class="px-3">
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-lg car-img">
+          <img class="img-fluid car-img mobile-hide " src="{{asset('/assets/car-sd.png')}}" alt="logo">
+        </div>
+        <div class="col-lg">
+          <div class="checker-box">
+            <div class="text-center">
+              <div class="pt-2">
+                <div class="Welcome-text-title">Selamat Datang</div>
+                <div class="Welcome-text-title">Calon Asisten Daskom</div>
+                <div class="Welcome-text-title">2022</div>
+                <!-- <span class="Welcome-text-title">
+                  <p>Selamat Datang</p>
+                  <p>Calon Asisten Daskom</p> 
+                  <p>2022</p> 
+                  </span> -->
+              </div>
+              <div class="pt-2">
+                <span class="tagline-home">
+                  #FindYourPathNeverBeAfraid
+                </span>
+              </div>
+              <div>
+                <span class="tagline-home">
+                  #LoopingForever
+                </span>
+              </div>
+            </div>
+            <div class="row">
+              <div class="row-lg">
+                <div class="d-flex justify-content-center pt-3">
+                  <a style="text-decoration: none" href="\ceklulus">
+                    <button class="home-button">
+                      Cek Kelulusan
+                    </button>
+                  </a>
+                </div>
+              </div>
+              @if($plotactive->isPlotActive==1)
```

Comment:
karena ini php jadi kalau buat comparison selalu biasain pake `===` ya fer, biar aman (biar ga kena celah keamanan namanya type juggling)
```suggestion
              @if($plotactive->isPlotActive===1)
```

Reply:
Ini utk semua if else kah bang?

Reply:
yoii bener fer